Urbuddi/src/main/java/com/urbuddi/pages/LeaveManagementPage.java

```diff
@@ -0,0 +1,104 @@
+package com.urbuddi.pages;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.time.temporal.ChronoUnit;
+
+import com.urbuddi.base.BasePage;
+
+public class LeaveManagementPage extends BasePage {
+	String LeaveManagementMenu = "css=[href*='leave_management'] li";
```

Comment:
can you check variable naming conventions in java and then change locator names as per the java standards @aramya2021